ee/orchestration/openapi/v2.yaml

```diff
@@ -659,8 +659,6 @@ components:
       type: object
       required:
         - triggerID
-        - workflowInstanceID
```

Comment:
Imply a breaking changes on the sdk. A simple value will become a pointer? Is this ok with the actual version of orchestration? @flemzord

Reply:
Is FCTL still compiling after this change ?